hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1813,7 +1813,7 @@ private void checkRemoteOMConfig(String remoteNodeId,
     }
 
     if (remoteOMConfig.getCurrentPeerList().contains(this.getOMNodeId())) {
-      throw new IOException("Remote OM " + remoteNodeId + " already contains " +
+      LOG.warn("Remote OM " + remoteNodeId + " already contains " +
```

Comment:
It is better to use log4j templates instead of string concatenation here

Reply:
Done